File: Java NetBeans IDE 8.0/1-Valores/src/valores/Operaciones.java
package valores;

import javax.swing.JOptionPane;

/**
 *
 * @author Benjamin
 */
public class Operaciones {
    private float n1,n2,n3;
    public void Inicilizar(){
        try {
            this.n1=Float.parseFloat(JOptionPane.showInputDialog("Ingresa primer valor: "));
            this.n2=Float.parseFloat(JOptionPane.showInputDialog("Ingresa segundo valor: "));
            this.n3=Float.parseFloat(JOptionPane.showInputDialog("Ingresa tercer valor: "));
        } catch (Exception e) {
            System.out.println("Error al ingresar datos, intenta de nuevo.");
        }
    }

    public void CalcularMayor(){
        if(n1>=n2 && n1>=n3){
            System.out.println("El numero mayor es "+n1);
        }
        else if(n2>=n1 && n2>=n3){
            System.out.println("El numero mayor es "+n2);
        }
        else if(n3>=n1 && n3>=n2){
            System.out.println("El numero mayor es "+n3);
        }
        else System.out.println("Numeros iguales");
    }
    public void CalcularMenor(){
        if(n1<=n2 && n1<=n3){
            System.out.println("El numero menor es "+n1);
        }
        else if(n2<=n1 && n2<=n3){
            System.out.println("El numero menor es "+n2);
        }
        else if(n3<=n1 && n3<=n2){
            System.out.println("El numero menor es "+n3);
        }
        else System.out.println("Numeros iguales");
    }
}

File: Python With PyCharm/1-Valores/Operaciones.py
n1=0
n2=0
n3=0
def Inicializacion():
    global n1,n2,n3
    try:
        n1=float(input("Ingresa el primer valor: "))
        n2=float(input("Ingresa el segundo valor: "))
        n3=float(input("Ingresa el tercer valor: "))
    except:
        if not type(n1) is float or not type(n2) is float or not type(n3) is float:
            raise TypeError("Solo se pueden ingresar datos numericos")

def CalcularMayor():
    if n1>=n2 and n1>=n3:
        print("El numero mayor es: ",n1)
    else:
        if n2>=n1 and n2>=n3:
            print("El numero mayor es: ",n2)
        else:
            if n3>=n1 and n3>=n2:
                print("El numero mayor es: ",n3)

def CalcularMenor():
    if n1<=n2 and n1<=n3:
        print("El numero menor es: ",n1)
    else:
        if n2<=n1 and n2<=n3:
            print("El numero menor es: ",n2)
        else:
            if n3<=n1 and n3<=n2:
                print("El numero menor es: ",n3)